_config.php

```diff
@@ -1,2 +1 @@
 <?php
-// Noop.
```

Comment:
This is here on purpose; the point is to reinforce the fact that this file is here and intentionally blank (hence the "noop", short for "no operation").